poll/public/handlebars/poll_results.handlebars

```diff
@@ -0,0 +1,43 @@
+<script id="poll-results-template" type="text/html">
+    <h3 class="poll-header">{{display_name}}</h3>
+    <div class="poll-question-container">{{{question}}}</div>
+    <ul class="poll-answers-results poll-results">
+    {{#each tally}}
+        <li class="poll-result">
+            <div class="poll-result-input-container">
+              <input id="answer-{{key}}" type="radio" disabled {{#if choice}}checked="True"{{/if}} />
+            </div>
+            {{#if any_img}}
+                <div class="poll-image result-image">
+                    <label for="answer-{{key}}" class="poll-image-label">
+                        {{#if img}}
+                          <img src="{{img}}" />
+                        {{/if}}
+                    </label>
+                </div>
+            {{/if}}
+            <div class="percentage-gauge-container">
+              <div class="percentage-gauge" style="width:{{percent}}%;">
+                  <label class="poll-answer-label" for="answer-{{key}}">{{{answer}}}</label>
```

Comment:
We allow to use Markdown in the answers, mainly to make it easy to add emphasis and links (well, to add phrasing content).  Markdown wraps all text in `<p>` elements, and this is how they end up in the label.

This is rather tricky to fix.  We can't turn the label into anything else, since it needs to be associated with the radio button, and we can't turn off Markdown without breaking backwards compatibility.  We might be able to filter out tags that are not phrasing content, but that would be a bit of effort to get done.

How important is it to get this fixed?  While this is invalid HTML, existing browsers don't seem to have a problem with it.  Will it be a problem for screen readers?